components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1252,6 +1297,93 @@ def port_val(port_type, value):
         except:
             self.ctx.out("OMERO.web not installed!")
 
+    def get_users_groups(self, args, iadmin):
```

Comment:
Since this method is related to `add_user_and_group_arguments`, it would make sense to move it to `UserGroupControl`.